docs/extending/creating_custom_dropdowns.md

```diff
@@ -94,15 +140,8 @@ Full list of options:
 
 |Name|Description|Required|
 |----|-----------|--------|
-|`container`|contains a reference to a DOM node where custom drop-down is initialized.|required|
-|`sourceInput`|contains a reference to a DOM node where the value of selected option is stored. Preferably, it should be a reference to a select input node.|required|
-|`itemsContainer`|contains a reference to a duplicated items container.|required|
-|`hasDefaultSelection`|contains a boolean value. If set to `true` the first option is selected as a placeholder or selected value.|optional|
-|`selectedItemTemplate`|contains a literal template string with placeholders for `value` and `label` data.|optional|
-
-In the code samples you can find 4 of 5 configuration options.
-Default template HTML code structure for missing `selectedItemTemplate` looks like this:
-
-```html
-    <li class="ibexa-custom-dropdown__selected-item" data-value="{{value}}">{{label}}<span class="${CLASS_REMOVE_SELECTION}"></span></li>
-```
+|`container`|Contains a reference to a DOM node where custom drop-down is initialized.|required|
+|`selectorSource`|Contains a reference |required|
+|`itemsContainer`|Contains a reference to a duplicated items container.|required|
+|`hasDefaultSelection`|Contains a boolean value. If set to `true` the first option will be selected as a placeholder or selected value.|optional|
+|`selectedItemTemplate`|Contains a literal template string with placeholders for `value` and `label` data.|optional|
```

Comment:
```suggestion\
```
We no longer use these three parameters

Reply:
@GrabowskiM which three you mean? :)

Reply:
itemsContainer, hasDefaultSelection and selectedItemTemplate